Orange/widgets/visualize/tests/test_owscatterplot.py

```diff
@@ -18,13 +18,17 @@ class TestOWScatterPlot(WidgetTest, WidgetOutputsTestMixin):
     def setUpClass(cls):
         super().setUpClass()
         WidgetOutputsTestMixin.init(cls)
+        cls.same_input_output_domain = False
 
         cls.signal_name = "Data"
         cls.signal_data = cls.data
 
     def setUp(self):
         self.widget = self.create_widget(OWScatterPlot)
 
+    def _compare_selected_annotated_domains(self, selected, annotated):
+        pass
```

Comment:
What is this?

Reply:
This disables an inapplicable test from the base class.

Hierarchical clustering, which has a similar situation, [replaces this](https://github.com/biolab/orange3/blob/master/Orange/widgets/unsupervised/tests/test_owhierarchicalclustering.py#L32) with checking that the domains have same variables and that the annotation appears only in one domain ... but I don't see much sense in testing this here, so I just disabled the test.

I now added a short comment explaining why the method is overridden.

Reply:
> I now added a short comment explaining why the method is overridden.

Thanks.